.github/workflows/publish_package_to_pypi.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Build and Publish Grater Expectations Package
+
+on:
+  release:
+    types: [published]
```

Comment:
Do we have anywhere a pipeline to publish the release or do we expect the devs to do it manually?

Asking this as I am trying to picture the new release flow:
- Tag a branch
- Pipeline A publishes it to testPypI
- Manually publish the release if tests are positive
- Pipeline B publishes to PyPi

Reply:
The flow is exactly as you described. the release is manual.

Reply:
love it